src/scripts/dungeons/Dungeon.ts

```diff
@@ -308,14 +308,17 @@ class Dungeon {
             // Handling Pokemon
             if (typeof enemy === 'string' || enemy.hasOwnProperty('pokemon')) {
                 let pokemonName: PokemonNameType;
+                let hideEncounter = false;
                 if (enemy.hasOwnProperty('pokemon')) {
                     pokemonName = (<DetailedPokemon>enemy).pokemon;
+                    hideEncounter = (<DetailedPokemon>enemy).options?.hide ? ((<DetailedPokemon>enemy).options?.requirement ? !(<DetailedPokemon>enemy).options?.requirement.isCompleted() : (<DetailedPokemon>enemy).options?.hide) : false;
                 } else {
                     pokemonName = <PokemonNameType>enemy;
                 }
                 const encounter = {
                     image: `assets/images/${(App.game.party.alreadyCaughtPokemonByName(pokemonName, true) ? 'shiny' : '')}pokemon/${pokemonMap[pokemonName].id}.png`,
                     shiny:  App.game.party.alreadyCaughtPokemonByName(pokemonName, true),
+                    hide: hideEncounter,
                     hidden: !App.game.party.alreadyCaughtPokemonByName(pokemonName),
```

Comment:
Using both hide and hidden is confusing. One of them need to change name

Reply:
@RedSparr0w Your system, any thoughts on this?

Reply:
We should probably change `hidden` to `caught` or `uncaught` as that is more inline with what the option actually does.
_(would also need to be changed on the boss encounter const too)_